cli-commands/pg/post/delete-firewall-rule.rb

```diff
@@ -1,11 +1,17 @@
 # frozen_string_literal: true
 
-UbiRodish.on("pg").run_is("delete-firewall-rule", args: 1, invalid_args_message: "rule id is required") do |ubid|
-  if ubid.include?("/")
-    raise Rodish::CommandFailure, "invalid firewall rule id format"
-  end
+UbiRodish.on("pg").run_on("delete-firewall-rule") do
+  options("ubi pg location/(pg-name|_pg-ubid) delete-firewall-rule id")
+
+  args 1, invalid_args_message: "rule id is required"
+
+  run do |ubid|
+    if ubid.include?("/")
+      raise Rodish::CommandFailure, "invalid firewall rule id format"
+    end
```

Comment:
Seeing this pattern, and that it is just applied to a subset (ie firewall rule above with cidr didn't have any format checks) makes me wonder if this should be a more universal pattern. Like maybe the `args` command could take a regex or something (as a blunt instrument in terms of what comes immediately to mind). Also in this particular case, there are presumably a lot of invalid values other than including a slash. Though I guess maybe anything that doesn't cause a client side error could potentially be delegated to the server for errors?

Reply:
Correct.  We are only checking for slash here to ensure that it will hit the same api route.